### DISCUSSION TITLE
pippidon clear animation displays one empty frame

### DISCUSSION THREAD

**Discussion by chayleaf** [ppy/osu#16928] [Q&A] [answered]
[here's](https://drive.google.com/file/d/1umgvfLtaCnfhGsIUmHJqkdbGcnEMosPS/view) my skin. During pippidon clear animation, one frame is always empty. This works properly on stable.

**Comment by bdach** [comment, marked as answer]
I've checked your skin and can reproduce no such thing, all frames are visible properly as far as I can tell. Can you attach a video of this?

**Reply by bdach** [reply]
The video linked above is entirely black...?

**Reply by chayleaf** [reply]
sorry, sent the wrong file on accident... https://youtu.be/iCIWER55AWY

**Reply by bdach** [reply]
I'm still at a loss. For reference, this is how the map and skin are looking for me.

https://user-images.githubusercontent.com/20418176/154862836-473134c1-2d84-4bfc-a923-65caca27044a.mp4

Anything useful in the logs, perhaps...?

**Reply by chayleaf** [reply]
you're right, it seems to work now... perhaps it was a skin migration bug in a past game version? Either way, I'm closing this
